### DISCUSSION TITLE
Do I need an NPM login if I only want to publish to GH registry?

### DISCUSSION THREAD

**Discussion by patrickgrey** [community/community#26200] [Actions]
<p>Noob question but I found the docs confusing when getting started (understand we are in beta). If I only want my packages to appear in GH registry, do I need the .npmrc file with NPM login details? When creating an Action I’m not sure how <strong>or if</strong> this ties in with </p>
<pre><code>NODE_AUTH_TOKEN: ${{secrets.GITHUB_TOKEN}}
</code></pre>
<p>Any clarification would be welcome.</p>

**Comment by chrispat** [comment, marked as answer]
<p>Yes you still need to authenticate npm to push to GitHub Packages.  The string below simply uses the Actions token as the password in the npmrc file.  </p>

**Comment by patrickgrey** [comment]
<p>Ah, thank you for answering question and clarifying how they work together!</p>
<p>Cheers!</p>
